<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectResDialog</class>
 <widget class="QDialog" name="SelectResDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select tile</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="mListWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mSelectBtn">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mSelectBtn</sender>
   <signal>pressed()</signal>
   <receiver>SelectResDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCancelBtn</sender>
   <signal>pressed()</signal>
   <receiver>SelectResDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
